src/main/java/org/folio/rest/impl/CirculationSettingsAPI.java

```diff
@@ -12,16 +12,23 @@
 import io.vertx.core.Context;
 import io.vertx.core.Handler;
 
+import static io.vertx.core.Future.succeededFuture;
+import static org.folio.rest.jaxrs.resource.CirculationSettingsStorage.PostCirculationSettingsStorageCirculationSettingsResponse.headersFor201;
+import static org.folio.rest.jaxrs.resource.CirculationSettingsStorage.PostCirculationSettingsStorageCirculationSettingsResponse.respond201WithApplicationJson;
+import static org.folio.rest.jaxrs.resource.CirculationSettingsStorage.PostCirculationSettingsStorageCirculationSettingsResponse.respond500WithTextPlain;
+
 public class CirculationSettingsAPI implements CirculationSettingsStorage {
 
   @Override
   public void postCirculationSettingsStorageCirculationSettings(String lang,
     CirculationSetting circulationSettings, Map<String, String> okapiHeaders,
     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-
```

Comment:
Please bring back the empty line. If method signature spans multiple lines, we add an empty line after it.

Reply:
done